modules/sql/src/main/scala/MagnitudeSystem.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.sql
+
+/** Minimal MagnitudeSystem classes to support reading from the enum
+  * tables.
+  */
+
+final case class MagnitudeSystem(override val toString: String)
+object MagnitudeSystem {
+  def fromString(s: String): MagnitudeSystem = MagnitudeSystem(s)
+}
```

Comment:
This doesn't feel right.

Reply:
You don't need the factory method here. I would just say `MagnitudeSystem(id: String)` and call it good. The only purpose is to tag it as something other than a `String`.